taxonomies/categories.txt

```diff
@@ -1964,44 +1964,44 @@ ru:Тоник
 zh:通宁水
 wikidata:en:Q237315
 
-<en:Diet beverages
+<en:Artificially sweetened beverages
 <en:Sodas
-en:Diet sodas, diet soft drinks
-de:Kalorienarme Erfrischungsgetränke, Kalorienreduzierte Erfrischungsgetränke, Diät-Erfrischungsgetränke
-es:Sodas light
+en:Artificially sweetened sodas, diet sodas, diet soft drinks
```

Comment:
I think we should discuss this on Slack in #taxonomies. I'd prefer to keep "Diet sodas" as the main name of the category.